build/updater.sh

```diff
@@ -2,8 +2,12 @@
 cd /data/project/query-builder-test/query-builder
 git reset --hard origin/master
 git pull
+export NODE_ENV=production
 npm install npm@6.14.8
 ./node_modules/.bin/npm install
 ./node_modules/.bin/npm run build
 rm -rf public_html/*
 cp -R dist/* ../public_html/
+cat <<EOT >> env.production.local
```

Comment:
Isn't this part needed before `./node_modules/.bin/npm run build`?

I feel like we could have one file `env.production` and that holds our production configuration. It would probably be read with `source env.production` or something similar?

Maybe at some later point we can enable a local override, when 3rd parties also want to use it themselves or we have some sensitive data here that shouldn't be in public version control.

Reply:
@micgro42  regarding the naming : (VUE_APP_WIKIDATA_API_URL), vue doesn't pick up env files unless they're prefixed with VUE_APP, as noted [here](https://cli.vuejs.org/guide/mode-and-env.html#environment-variables)

Reply:
> @micgro42 regarding the naming : (VUE_APP_WIKIDATA_API_URL), vue doesn't pick up env files unless they're prefixed with VUE_APP, as noted [here](https://cli.vuejs.org/guide/mode-and-env.html#environment-variables)

Ok, that makes sense. Then yes, let's prefix the names with that.